src/containers/Network/Nodes.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react'
+import axios from 'axios'
+import { useTranslation } from 'react-i18next'
+import { useQuery } from 'react-query'
+import NetworkTabs from './NetworkTabs'
+import Map from './Map'
+import NodesTable from './NodesTable'
+import Log from '../shared/log'
+import { FETCH_INTERVAL_NODES_MILLIS, localizeNumber } from '../shared/utils'
+import { useLanguage } from '../shared/hooks'
+
+export const Nodes = () => {
+  const language = useLanguage()
+  const { t } = useTranslation()
+  const [nodes, setNodes] = useState<any>([{}])
+  const [locations, setLocations] = useState([])
+  const [unmapped, setUnmapped] = useState(0)
+
+  useQuery('fetchNodesData', async () => fetchData(), {
+    refetchInterval: FETCH_INTERVAL_NODES_MILLIS,
+  })
+
+  const fetchData = () => {
+    axios
+      .get('/api/v1/nodes')
+      .then((resp) => {
+        const nodesWithLocations = resp.data.filter(
+          (node: any) => 'lat' in node,
+        )
+        setNodes(resp.data)
+        setUnmapped(resp.data.length - nodesWithLocations.length)
+        setLocations(nodesWithLocations)
+      })
+      .catch((e) => Log.error(e))
+  }
+
+  return (
+    <div className="network-page">
+      {
+        // @ts-ignore - Work around for complex type assignment issues
```

Comment:
This is caused by the `connect` function in Map jacking up its type.  It claims there are no props on the component

Reply:
Can we fix this by modifying the connect function in Map?

Reply:
I tried doing that really quick.  It needs to be made typescript and there were a lot of errors when i did that.  Also we are using such and old version of redux i dont know if they type can be corrected.

Reply:
Hmm I guess it can be a separate refactor ticket then